docs/core/v3.0.rst

```diff
@@ -441,38 +441,57 @@ Node names
 
 The root node does not have a name and is the empty string ``""``.
 Except for the root node, each node in a hierarchy must have a name,
-which is a string of characters. To ensure consistent behaviour
-across different storage systems, the following constraints apply to
+which is a string of characters. The following constraints apply to
```

Comment:
I would say "Unicode code points" rather than "characters" here, and then you don't need to specify "unicode code point sequence" below.